definitions/json/AuthorisationCaseField.json

```diff
@@ -1,4 +1,34 @@
 [
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "scannedDocuments",	
+    "UserRole": "caseworker-ET-pcqextractor",
```

Comment:
This role (caseworker-ET-pcqextractor)  is only needed if we need to consolidate Pcq Id . If you are sending ccdCaseId all the time then consolidation is not needed and all the entries using this role can be removed.